src/adapter/search-request-adapter/__tests__/search-params.tests.ts

```diff
@@ -162,7 +164,6 @@ describe('Pagination adapter', () => {
     const searchParams = adaptSearchParams({
       ...DEFAULT_CONTEXT,
       pagination: { paginationTotalHits: 4, page: 0, hitsPerPage: 6 },
-      finitePagination: false,
```

Comment:
Can you explain that?

Reply:
Not very much related to the PR, I wanted to remove these unnecessary lines.